ios/Exponent/Versioned/Modules/Api/EXContacts.m

```diff
@@ -31,12 +31,33 @@ @implementation EXContacts
 
   // always include id, name, firstName, middleName, lastName, company, jobTitle
   NSMutableSet *fieldsSet = [NSMutableSet setWithArray:options[@"fields"]];
-  [fieldsSet addObjectsFromArray:@[@"id", @"name", @"firstName", @"middleName", @"lastName", @"company", @"jobTitle"]];
+  [fieldsSet addObjectsFromArray:@[
+                                   @"id",
+                                   @"contactType",
+                                   @"name",
+                                   @"firstName",
+                                   @"middleName",
+                                   @"lastName",
+                                   @"previousLastName",
+                                   @"nickname",
+                                   @"company",
+                                   @"jobTitle",
+                                   @"department",
+                                   @"imageAvailable"
+                                   ]];
 
   NSArray *keysToFetch = [self _contactKeysToFetchFromFields:fieldsSet.allObjects];
+  NSString *id = options[@"id"];
```

Comment:
Can we call this something else? Having a variable called `id` is confusing in objective-c